src/main/resources/i18n/messages.properties

```diff
@@ -144,7 +144,7 @@ clan.leader=Leader
 clan.clanMembers=Members
 clan.messageLeader=Message clan leader
 clan.visitPage=Visit clan website
-chat.privateMessage.playerLeft={0} is now offline
+chat.privateMessage.playerLeft={0} is now offline and left chat
```

Comment:
The problem is if you are chatting and the user is your friend what should be quite common then two Notifications are shown both containing the same message.  It therfore looks like a bug. Acctually i did not want to commit that change to this branch, but yeah. So we either change the messanges or make clear only one is shown in this particular case....